packages/plugins/plugin-appsflyer/src/AppsflyerPlugin.tsx

```diff
@@ -35,4 +85,41 @@ export class AppsflyerPlugin extends DestinationPlugin {
     track(event);
     return event;
   }
+
+  conversionCallback = () => {
+    appsFlyer.onInstallConversionData((res) => {
+      const isFirstLaunch = JSON.parse(res?.data?.is_first_launch);
```

Comment:
Why does it have to parse this variable? Is it a string instead of a boolean or something? can't we use it directly?

Reply:
Asking because `JSON.parse` might be an expensive operation

Reply:
A good practice would also be to unwrap all the values you need from `res?.data` as in:
```
const { af_status,  media_source, is_first_launch } = res?.data;
```

You can also assign them default values when destructuring.